external/fv3fit/setup.py

```diff
@@ -16,7 +16,7 @@
     "typing_extensions>=3.7.4.3",
     "dacite>=1.6.0",
     "wandb[media]>=0.12.1",
-    "pace-util>=0.7.0",
+    "pace-util>=0.9.0",
```

Comment:
bit of a nitpick, but does fv3fit really require 0.9.0? It's best to try and leave the requirements in specific packages as loose as possible. Either way, 0.9.0 will be used since it is specified as a requirement in pip-requirements.txt.

Reply:
Nope, that's fine

Reply:
I think the minimal update is `pip-requirements.txt` (for the fv3net env, given that this upgrade to pace-util is useful for avoiding future issues in dev work) and `workflows/prognostic_c48_run/requirements.in` (since the prognostic run image doesn't pick up `pip-requirements.txt`.

Reply:
Sounds good. `make lock_pip` should pick up on `workflows/prognostic_c48_run/requirements.in` for compiling `constraints.txt` and so I think it would (or should) work to just update there and not in `pip-requirements.txt`, but doing it in both is okay too.